### ISSUE TITLE
Custom class DataFrames (plural) to save a list of dataframes to multiple sheets in a spreadsheet?

### ISSUE THREAD

**Issue by baogianghoangvu** [baogianghoangvu/Pandas-Google-Sheets#1]
Sounds convenient but is it?
Naming individual df might be an issue or not?

Could be something like:
```python
DataFrames.to_googlesheets(titles=..., url=..., credentials=...)

# to be consistent with the standard single dataframe API
DataFrame.to_googlesheets(title=..., url=..., credentials=...)
```

**Comment by baogianghoangvu** [comment]
Also check out pd.readexcel to see how they handle multiple df (i.e. Sheets)

**Comment by hanh-nguyen** [comment]
sheets share the main part of the url but with different gid, so we need to have two parameters to read multiple sheets.

**Comment by baogianghoangvu** [comment]
> sheets share the main part of the url but with different gid, so we need to have two parameters to read multiple sheets.

Yes thank you for the comment, helps me realize this is part of a bigger discussion on how to design the interface.

Also brings to attention the fact that the current `read_googlesheets` implementation in #9 only deals with URL with both `sheet_id` and `spreadsheet_id`.
